src/flamegpu/io/XMLStateWriter.cu

```diff
@@ -0,0 +1,435 @@
+
+/**
+ * @file
+ * @author
+ * @date
+ * @brief
+ *
+ * \todo longer description
+ */
```

Comment:
This seems pointless? I assume it was in the .cpp version of this file and its just been retained (git's decided they are different enough to not show a useful diff).

Reply:
yes